src/lib/browser-extension-signing-manager.ts

```diff
@@ -77,14 +77,37 @@ export class BrowserExtensionSigningManager implements SigningManager {
     this._accountTypes = accountTypes;
   }
 
+  private isAccountTypeValid(account: InjectedAccount): boolean {
+    // If account has no type, it's valid
+    if (!account.type) {
+      return true;
+    }
+
+    // If no account types specified but account is ethereum, it's invalid
+    if (!this._accountTypes && account.type === 'ethereum') {
```

Comment:
If someone passes `ethereum` in it will still cause an error as we still try to decode and encode in in changeAddressFormat https://github.com/PolymeshAssociation/browser-extension-signing-manager/blob/c5e88793d776df05a0e321650dba1e44312b7f75/src/utils/index.ts#L13

If we are going to allow ethereum when mapping accounts we should skip converting to prevent the error.

Reply:
I didn't like to filter out `ethereum` even if it is explicitly sent but maybe the solution is to avoid it completely. 
What do you think? 
Option A -> Remove `ethereum` from the supported types by creating our own `KeypairType` 
Option B -> Skip converting `ethereum` addresses. I'm not sure if it would be useful anyways.